core/src/main/java/com/predic8/membrane/core/interceptor/oauth2/processors/CertsEndpointProcessor.java

```diff
@@ -35,15 +42,25 @@ public boolean isResponsible(Exchange exc) {
     }
 
     @Override
-    public Outcome process(Exchange exc) throws Exception {
+    public Outcome process(Exchange exc) {
         String accessTokenJWKIfAvailable = authServer.getTokenGenerator().getJwkIfAvailable();
         String idTokenJWK = authServer.getJwtGenerator().getJwk();
 
         String jwks = "{\"keys\": [ " + idTokenJWK +
                 (accessTokenJWKIfAvailable != null ? "," + accessTokenJWKIfAvailable : "") +
                 "]}";
 
-        exc.setResponse(Response.ok().contentType(APPLICATION_JSON_UTF8).body(jsonBeautifier.beautify(jwks)).build());
+        try {
+            exc.setResponse(ok().contentType(APPLICATION_JSON_UTF8).body(jsonBeautifier.beautify(jwks)).build());
+        } catch (IOException e) {
+            log.error(e.getMessage(), e);
```

Comment:
`e.getMessage()` is already printed, because `e` is passed to `log.error(...)` in the `Throwable` parameter.